falcon/response.py

```diff
@@ -246,9 +246,21 @@ def set_cookie(self, name, value, expires=None, max_age=None,
 
     def unset_cookie(self, name):
         """Unset a cookie from the response
+        
+        Note:
+            This will instruct the browser to expire its copy of the cookie
+            if it has previously been set.
         """
-        if self._cookies is not None and name in self._cookies:
-            del self._cookies[name]
+        if self._cookies is None:
+            self._cookies = SimpleCookie()
+        if name not in self._cookies:
+            self._cookies[name] = ""
+        # SimpleCookie apparently special cases the expires attribute to
```

Comment:
Nitpick: Add a blank line before the start of the comment to aid in readability. Also, start the comment with ` NOTE(Freezerburn): `.